--- a/src/scripts/fssrgen.fsx	2024-07-03 19:52:18.352525457 +0100
+++ b/src/scripts/fssrgen.fsx	2024-07-03 19:52:29.680734013 +0100
@@ -331,7 +331,7 @@
 
 let RunMain(filename, outFilename, outXmlFilenameOpt, projectNameOpt) =
     try
-        let justfilename = System.IO.Path.GetFileNameWithoutExtension(filename)
+        let justfilename: string = System.IO.Path.GetFileNameWithoutExtension(filename: string)
         if justfilename |> Seq.exists (fun c -> not(System.Char.IsLetterOrDigit(c))) then
             Err(filename, 0, sprintf "The filename '%s' is not allowed; only letters and digits can be used, as the filename also becomes the namespace for the SR class" justfilename)
 
